blog/_posts/2018-07-09-welcome-to-team-popcorn.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Meet Team Popcorn
+layout: post
+created_at: Mon July 09 2018
+permalink: blog/2018-07-09-welcome-to-team-popcorn.md
```

Comment:
Hi @sleepypioneer! Thanks for the PR. I just have minor comments. You don't need markdown file extension (.md) for 'permalink'.